frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -160,7 +160,7 @@ action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-
 action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.diverged-from.and-older-than-remote=diverged from (& older than) its remote
 action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.in-sync-to-remote=in sync with its remote
 action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.untracked=untracked
-action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.unknown=in unknown relation ''{0}'' to its remote
+action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.unknown=in unknown syncToRemoteStatus ''{0}'' to its remote
```

Comment:
Let's leave `relation` here, as it was (that's just a message for a user, we can be a little bit informal here, as compared to the naming we use in the code itself)